src/main/java/org/assertj/core/internal/Classes.java

```diff
@@ -206,6 +207,22 @@ public void assertIsProtected(AssertionInfo info, Class<?> actual) {
     }
   }
 
+  /**
+   * Verifies that the actual {@code Class} is package-private.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the "actual" {@code Class}.
+   * @throws AssertionError if {@code actual} is {@code null}.
+   * @throws AssertionError if the actual {@code Class} is not protected.
```

Comment:
Whoops I missed this before 🤦‍♂ it should be `package-private` instead of `protected`. We can adjust it during the merge